assets/dev/js/frontend/tools/stretch-element.js

```diff
@@ -16,7 +16,7 @@ module.exports = elementorModules.ViewModule.extend( {
 		};
 	},
 
-	stretch() {
+	stretch( useCssVariables = false ) {
```

Comment:
Add this config to `getDefaultSettings()` in this file.

Reply:
Updated.